warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ContentIndexingColumnBasedHandler.java

```diff
@@ -666,8 +672,9 @@ protected void indexListEntries(final NormalizedContentInterface nci, boolean in
         String content = nci.getIndexedFieldValue();
         
         String[] tokens = StringUtils.split(content, listDelimiter);
-        
+        int position = 0;
         for (String token : tokens) {
+            String trimmedToken = StringUtils.trim(token);
```

Comment:
Lets skip over empty tokens

Reply:
This could be an ordered list and that position is left empty on purpose. Is 25,,30 and valid list?

Reply:
I do not want to index empty values.  This would be just wrong.

Reply:
Fixed in next commit.